src/lava/magma/core/learning/constants.py

```diff
@@ -85,3 +86,10 @@
     str_symbols.Y2: 1,
     str_symbols.Y3: 2
 }
+
+
+class GradedSpikeCfg(IntEnum):
+    DEFAULT = 0
```

Comment:
I wonder if this should have a descriptive name also, for example, `UPDATE`. The fact that it is the default should be reflected in the default value where it is instantiated.

Reply:
Addressed this.